company-profiles/turing.md

```diff
@@ -0,0 +1,38 @@
+# Turing
+
+## Company blurb
+
+We are a data-science-driven deep jobs platform helping companies spin up their engineering teams in the cloud at the push of a button. Based in Palo Alto, California, we are a fully remote company of 900+ people who help connect world-class remote software engineers with world-class companies.
+
+ 
+We make the remote hiring journey easy and rewarding for both companies and developers. With Turing, companies can hire pre-vetted, Silicon Valley-caliber remote software talent across 100+ skills in 3-5 days. We also democratize opportunities for remote developers from around the world by offering them high-quality software jobs with top US firms.
+
+ 
+Our Intelligent Talent Cloud uses AI to source, vet, match, and manage over 2 million developers worldwide. This, in turn, helps organizations save valuable time and resources as they build their dream engineering team in a matter of days.
+
+
+## Remote status
+
+100 % remote jobs .
+
+## Region
+
+Worldwide
+
+## Company technologies
```

Comment:
These aren't technologies so either remove this section completely or edit to match programming languages or similar.

Reply:
I have removed the technologies section.